pkg/model/crd.go

```diff
@@ -566,6 +566,20 @@ func (r *CRD) PrintSyncedColumn() bool {
 	return r.cfg.ResourceDisplaysSyncedColumn(r.Names.Camel)
 }
 
+func (r *CRD) setAdditionalPrinterColumns(additionalColumns []*ackgenconfig.AdditionalColumnConfig) {
+	r.additionalPrinterColumns = []*PrinterColumn{}
+
+	for _, additionalColumn := range additionalColumns {
+		printerColumn := &PrinterColumn{}
+		printerColumn.Name = additionalColumn.Name
+		printerColumn.JSONPath = additionalColumn.JSONPath
+		printerColumn.Type = additionalColumn.Type
+		printerColumn.Priority = additionalColumn.Priority
+		printerColumn.Index = additionalColumn.Index
+		r.additionalPrinterColumns = append(r.additionalPrinterColumns, printerColumn)
+	}
+}
+
```

Comment:
The allocation on L570 overrides the existing printerColumns previously added in `model.go` L284 and L236. Removing L570 should fix the code and the unit tests